server/enclave/ecalls.cpp

```diff
@@ -26,6 +26,18 @@ if (!oe_is_outside_enclave((ptr), size)) {                \
 }                                                         \
 }
 
+// Defined in modelaggregator.conf
+static const int MAX_TCS = 32;
+
+// Ciphertext, IV, and tag are required for decryption
+static const size_t ENCRYPTION_METADATA_LENGTH = 3;
+
+// Global variables stored for threading
+static vector<map<string, vector<double>>> g_accumulator;
```

Comment:
Maybe you can also rename these global variables to `global_*` instead of `g_*` for clarity?

Reply:
Using `g_*` was a popular way to do it in many style guides I saw, such as: https://users.ece.cmu.edu/~eno/coding/CppCodingStandard.html#gnames

Reply:
Oh perfect, this is great then!